packages/react-instantsearch-hooks/README.md

```diff
@@ -618,6 +618,92 @@ function RefinementList(props) {
 }
 ```
 
+### `useSortBy`
+
+> `(props: UseSortByProps) => SortByRenderState`
+
+Hook to [sort by](https://www.algolia.com/doc/api-reference/widgets/sort-by/js/) specified indices.
```

Comment:
Not sure about the wording I used here, so feel free to suggest something that would be a better match.

Reply:
I think this wording is fine, otherwise the short description of the docs? 

> The sortBy widget displays a list of indices, allowing a user to change the way hits are sorted (with replica indices). Another common use case is to let the user switch between different indices.
> For this widget to work, you must define all indices that you pass down as options as replicas of the main index.

Reply:
It's fine for now indeed!